kernelci/config/data.py

```diff
@@ -0,0 +1,74 @@
+import yaml
```

Comment:
Missing copyright and license statement.

Reply:
Corrected